ee/api/conversation.py

```diff
@@ -151,6 +159,7 @@ def create(self, request: Request, *args, **kwargs):
             trace_id=serializer.validated_data["trace_id"],
             session_id=request.headers.get("X-POSTHOG-SESSION-ID"),  # Relies on posthog-js __add_tracing_headers
             mode=AssistantMode.ASSISTANT,
+            billing_context=serializer.validated_data.get("billing_context"),
```

Comment:
Question: `AssistantConversationRunnerWorkflowInputs` doesn't seem to have `billing_context` attribute though? How does it work?